src/protocols/bitcoin/libs/BitcoinAdapter.ts

```diff
@@ -153,6 +153,7 @@ export class BitcoinAdapter extends BaseProtocolAdapter {
     };
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Isn't it used?

Reply:
True. Leftover that needs to be removed.